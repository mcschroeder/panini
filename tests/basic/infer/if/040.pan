
f40 : {v:int|?}
f40 = let b = true in if b then 1 else 2

f41 : {v:int|?}
f41 = let b = false in if b then 1 else 2
