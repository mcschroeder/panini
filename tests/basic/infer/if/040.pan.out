f40 : {v:ℤ | v = 1}
f41 : {v:ℤ | v = 2}